docs/l4-l7.rst

```diff
@@ -479,3 +515,38 @@ Dynamic Endpoint Discovery functionality is provided by F5ACIServiceCenter and h
 To view pool members after share node option is disabled for an existing partition user need to sync endpoint manually to BIG-IP by Application inventory → Partition → Sync EP icon
 
 ------
+
+**Q.Why older partitions that no longer exist on the BIG-IP are still retrieved by the AS3 endpoint?**
```

Comment:
Why are older partitions that no longer exist on the BIG-IP still retrieved by the AS3 endpoint?